analyzers/tests/SonarAnalyzer.UnitTest/Rules/InfiniteRecursionTest.cs

```diff
@@ -29,9 +29,21 @@ namespace SonarAnalyzer.UnitTest.Rules
     public class InfiniteRecursionTest
     {
         [TestMethod]
-        public void InfiniteRecursion() =>
+        public void InfiniteRecursionSonarCfg() =>
             Verifier.VerifyAnalyzer(
                 @"TestCases\InfiniteRecursion.cs",
+                new InfiniteRecursion(true),
+#if NETFRAMEWORK
+                ParseOptionsHelper.FromCSharp8,
+                NuGetMetadataReference.NETStandardV2_1_0);
+#else
+                ParseOptionsHelper.FromCSharp8);
```

Comment:
That was not easy, since I had to change other test cases as well (like expression like returns, nullable stuff and so on). The reason - is that when you run this test locally it starts with C# 5

Reply:
You could have temporarily updated your local test framework to use the latest version instead of the oldest for this purpose.